.solhint.json

```diff
@@ -18,9 +18,9 @@
     "imports-on-top": "error",
     "ordering": "error",
     "visibility-modifier-order": "error",
-    "code-complexity": ["error", 9],
+    "code-complexity": ["error", 11],
     "function-max-lines": ["error", 80],
-    "max-line-length": ["error", 150],
+    "max-line-length": ["error", 170],
```

Comment:
can just lint with line breaks :)

Reply:
my current linter puts it all in same line and I was being lazy, clearly :)

Reply:
reverted this